apps/src/lib/ui/simpleSignUp/workshop/WorkshopStudentEnrollPage.tsx

```diff
@@ -12,6 +12,7 @@ const WorkshopStudentEnrollPage: React.FunctionComponent = () => (
       <AccountBanner
         heading={i18n.accountNeedTeacherAccountWelcomeBannerHeaderLabel()}
         desc={i18n.accountNeedTeacherAccountWelcomeBannerHeaderDesc()}
+        showLogo={true}
```

Comment:
This page still looks good:

<img width="1060" alt="Screenshot 2024-08-06 at 9 05 01 AM" src="https://github.com/user-attachments/assets/b25695ad-e469-4142-a15c-27fbb6be3a5f">